test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -253,11 +255,10 @@ public void testRowToText_specialCharacters() throws Exception {
 
     ResultSet rs = executeQueryAndNext("select * from data");
     resgen.generateResponse(rs, response);
-    String golden =
-        (is(MYSQL) ? "id,name,quote\n"
-                   : "ID,NAME,QUOTE\n")
+    String table = is(SQLSERVER) ? ",," : "DATA,DATA,DATA";
```

Comment:
This is still going to break on MySQL, where the table name comes back in lower case, depending on the filesystem used.